.gitignore

```diff
@@ -80,4 +80,8 @@ yarn-error.log
 public
 
 # Mac files
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Vim
+*.vim
```

Comment:
Please drop the commit adding these changes and force-push to  your branch. This is not necessary to add here and can be easily not committed.

Reply:
Done!